core/src/main/scala/akka/persistence/cassandra/snapshot/CassandraSnapshotCleanup.scala

```diff
@@ -20,22 +19,14 @@ trait CassandraSnapshotCleanup extends CassandraStatements {
   def session: CassandraSession
   private[akka] implicit val ec: ExecutionContext
 
-  private lazy val deleteRetryPolicy = new LoggingRetryPolicy(new FixedRetryPolicy(snapshotConfig.deleteRetries))
+  def preparedDeleteSnapshot: Future[PreparedStatement] =
+    session.prepare(deleteSnapshot)
 
-  def preparedDeleteSnapshot =
-    session
-      .prepare(deleteSnapshot)
-      .map(_.setConsistencyLevel(snapshotConfig.writeConsistency).setIdempotent(true).setRetryPolicy(deleteRetryPolicy))
+  def preparedDeleteAllSnapshotsForPid: Future[PreparedStatement] =
+    session.prepare(deleteAllSnapshotForPersistenceId)
 
-  def preparedDeleteAllSnapshotsForPid =
-    session
-      .prepare(deleteAllSnapshotForPersistenceId)
-      .map(_.setConsistencyLevel(snapshotConfig.writeConsistency).setIdempotent(true).setRetryPolicy(deleteRetryPolicy))
-
-  def preparedDeleteAllSnapshotsForPidAndSequenceNrBetween =
-    session
-      .prepare(deleteAllSnapshotForPersistenceIdAndSequenceNrBetween)
-      .map(_.setConsistencyLevel(snapshotConfig.writeConsistency).setIdempotent(true).setRetryPolicy(deleteRetryPolicy))
+  def preparedDeleteAllSnapshotsForPidAndSequenceNrBetween: Future[PreparedStatement] =
+    session.prepare(deleteAllSnapshotForPersistenceIdAndSequenceNrBetween)
```

Comment:
What about the retry policy in these, where is that now from?

Reply:
The profile :)

Reply:
*then standard answer* ;-)

Reply:
What's the defaults for retries? I think we had some support case where we saw some really long retry timeouts (> 30sec). Anything we need to adjust? It's not good with too long pending writings because then circuit breakers may trigger before the write is completed. Could even result in new actor starting on another node and not seeing the write if it completes after the recovery.

Reply:
I just have not seen it set in the profile here, hence the clarification.

Reply:
> I just have not seen it set in the profile here, hence the clarification.

The only thing we override in our profile is the consistency as we want the default to be QUORUM rather than LOCAL_ONE. Everything else is up to the user to override in the profile.

Reply:
> What's the defaults for retries? I think we had some support case where we saw some really long retry timeouts (> 30sec). Anything we need to adjust? It's not good with too long pending writings because then circuit breakers may trigger before the write is completed. Could even result in new actor starting on another node and not seeing the write if it completes after the recovery.

The driver has simplified this and provides no alternative retry policy, but it is still pluggable. 

The default retries once, and only if enough replicas responded but didn't return the data (they start off returning a hash). I think this is a fine default for us and users can plug in a different retry policy if needed